### PR TITLE
Updated instructions (more containers, s3cmd, whitespacing)

### PR DESCRIPTION
**Pull request by pontus**

### CODE REVIEW

File: user-guide.md [hunk 1]
```diff
@@ -82,9 +151,11 @@ source env/bin/activate
 .\env\Scripts\activate
 ```
 ​
-## Installing the encryption tool
+### Installing the crypt4gh encryption tool
 ​
-The next step is to install [crypt4gh](https://github.com/EGA-archive/crypt4gh), by running the following command:
+The next step is to install
+[crypt4gh](https://github.com/EGA-archive/crypt4gh), by running the
+following command:
```
Comment (jbygdell) [inline user-guide.md:158]: Another option if a static binary is preferred, it to use the GO version available from here: https://github.com/elixir-oslo/crypt4gh/releases/latest

File: user-guide-bianca.md [hunk 2]
```diff
@@ -1,29 +1,58 @@
...
+**Note**: If you already know how to move files to bianca, you can
+skip the details for this part.
+
+You can move the singularity image to Bianca using wharf. The guide
+for transferring files from and to Bianca can be found
+[here](https://www.uppmax.uu.se/support/user-guides/transit-user-guide/).
+
+In the following examples, replace
+`ega-crypt4gh_crypt4gh-keygen-1.0.0.sif` with `crypt4gh-1.0.0.sif` if
+you opted for downloading with `wget` (alternativly you can rename it with
```
Comment (dbampalikis) [inline user-guide-bianca.md]:
```suggestion
you opted for downloading with `wget`. Alternatively you can rename it with 
```

File: user-guide-bianca.md [hunk 3]
```diff
@@ -1,29 +1,58 @@
...
 Navigate to the folder you want to pull the images.
+
 * To download the crypt4gh image run the following command in the terminal:
+
 ```bash
 singularity pull docker://nbisweden/ega-crypt4gh:crypt4gh-1.0.0
 ```
-Now you should be able to see a file with `.sif` extensions in the folder when running the `ls` command.
+
+Alternative, you can download a ready made singularity image with:
```
Comment (dbampalikis) [inline user-guide-bianca.md]:
```suggestion
Alternatively, you can download a ready made singularity image with:
```

File: user-guide.md [hunk 4]
```diff
@@ -2,15 +2,66 @@
...
+standard. ​
+
 # Requirements
-​
-The easiest way to encrypt files is with the **crypt4gh** command line tool. That tool requires **Python 3.6** or newer and **virtualenv**. For uploading files we recommend **s3cmd** for linux and osx based systems and **s3Express** for Windows systems (**s3cmd** might work on windows as well).
-​
-## Check if Python is installed
 
-To check whether Python is already installed in your machine, you can run the following command in the terminal:
+Here we'll outline procedure for encrypting and uploading using tools such as
```
Comment (dbampalikis) [inline user-guide.md]:
```suggestion
Here we'll outline the procedure for encrypting and uploading using tools such as
```

File: user-guide.md [hunk 5]
```diff
@@ -135,35 +215,61 @@ which should return something like
...
+command in order to encrypt the file (if you want to use a secret key 
+you have):
+
 ​
 ```bash
 crypt4gh encrypt --sk <my-key>.sec --recipient_pk ega-se.key < <my-file> > <my-encypted-file>.c4gh
 ```
-​
+
+If you don't have a secret key, just remove that part:
```
Comment (dbampalikis) [inline user-guide.md]:
```suggestion
If you don't have a secret key, just remove that part and run:
```

**Review by dbampalikis — APPROVED** [review]

**Pull request merged**